tools/legacy/ObjectiveC/Legacy_ObjectiveC.csproj

```diff
@@ -25,6 +25,7 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <GeneratePkgDefFile>false</GeneratePkgDefFile>
+    <DeployExtension>False</DeployExtension>
```

Comment:
no I was seeing it locally and was upset. 

---
In reply to: [107544404](https://github.com/Microsoft/WinObjC/pull/2300#discussion_r107544404) [](ancestors = 107544404)